build.gradle.kts

```diff
@@ -1,6 +1,7 @@
 plugins {
 	id("io.github.gradle-nexus.publish-plugin") version "1.0.0"
 	id("io.gitlab.arturbosch.detekt").version(Dependencies.detektVersion)
+	id("org.jetbrains.dokka") version "1.4.30"
```

Comment:
Is that tied to the Kotlin version? If yes, could you use a constant/reference here?

Reply:
No but it should be moved to `buildSrc`. Forgot that